### ISSUE TITLE
Akka.Remote generated types for Protobuf have no namespace and collide with framework types

### ISSUE THREAD

**Issue by julienadam** [akkadotnet/akka.net#2443]
In [src/core/Akka.Remote/Proto/Wireformats.cs](https://github.com/akkadotnet/akka.net/blob/dev/src/core/Akka.Remote/Proto/Wireformats.cs), types are generated without a namespace.

Among them is CommandType which has the same name as System.Data.CommandType. This causes build issues like this :

``` csharp
using System.Data;
class Foo
{
    CommandType t = CommandType.StoredProcedure; // <-- Doesn't compile
}
```

Generated types should be in a namespace to avoid these name collisions. It shouldn't affect serialization, unless full type names are also serialized and / or used for serialization.

**Comment by Danthar** [comment]
Its probably an oversight on our end that these are not in a proper namespace. We can enable this in our proto compile step. But im not sure if this would break backward compat. @akkadotnet/core Can someone comment on that ?

**Comment by Aaronontheweb** [comment]
@Danthar these types are marked as `public`  but they're not really exposed in any meaningful way outside of the Akka.Remote built-in message type serializer. We could re-run the protobuf compiler with a namespace setting to correct this. Might have to stick a shim into the Akka.Remote serializers to account for the namespace difference but I don't think it'd be a catastrophic change.

**Comment by Aaronontheweb** [comment]
Resolved by #2459
